packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -966,6 +966,149 @@ describe.each([
     })
   })
 
+  !isInternal &&
+    describe("datetime - time only", () => {
+      const T_1000 = "10:00"
+      const T_1045 = "10:45"
+      const T_1200 = "12:00"
+      const T_1530 = "15:30"
+      const T_0000 = "00:00"
+
+      const UNEXISTING_TIME = "10:01"
+
+      beforeAll(async () => {
+        await createTable({
+          time: { name: "time", type: FieldType.DATETIME, timeOnly: true },
```

Comment:
Given the trouble we've had with nulls, I think it would be worth adding a row with a null time to make sure it works as expected.

Reply:
That said, https://github.com/Budibase/budibase/pull/13728 fixes a bunch of null problems so you may wish to wait until that goes in first.

Reply:
Done. I actually needed to fix some stuff related to pg 🤷🏽